UITests/BaseTests/BaseBusinessTest.swift

```diff
@@ -0,0 +1,35 @@
+import XCTest
+
+class BaseBusinessTests: XCTestCase {
```

Comment:
I think we can get rid of this class, either move out the setupStub for business user token or maybe pass in a parameter to control the flow instead.

Reply:
> I think we can get rid of this class, either move out the setupStub for business user token or maybe pass in a parameter to control the flow instead.

rewrote business and individual base classes to one base class